### ISSUE TITLE
Bug in `get_multi_pq_info` where `match_iou` is not used 

### ISSUE THREAD

**Issue by yiheng-wang-nv** [TissueImageAnalytics/CoNIC#15]
In this line:
https://github.com/TissueImageAnalytics/CoNIC/blob/9c52bd95dc284289e01de3d023175437b074aa15/metrics/stats_utils.py#L47

match_iou should be used.

**Comment by vqdang** [comment]
You are correct that it should be delegated down. In this challenge, It won't affect much as we usually use 0.5 anyways (default param for `get_pq`)
